geoportailv3/static/js/maincontroller.js

```diff
@@ -442,8 +471,7 @@ app.MainController.prototype.createMap_ = function() {
     logo: false,
     controls: [
       new ol.control.Zoom({zoomInLabel: '\ue032', zoomOutLabel: '\ue033'}),
-      new ol.control.ZoomToExtent({label: '\ue01b',
-        extent: this.defaultExtent_}),
+      new app.MainController.ZoomToExtent3d(this.defaultExtent_, () => this.ol3dm_),
```

Comment:
Name is confusing as it's both for 2d and 3d.
Also the control generally depends on the cesiumManager extent.
I would postpone adding of this control after `old3dm` creation, and pass as argument `old3dm` `cameraExtentInRadians`.

Reply:
done